templates/startpage.html.twig

```diff
@@ -42,11 +42,102 @@
+            </p>
+
+            <h4>Reproduce bugs</h4>
+            <p>
+                One thing that is super helpful for maintainers is reproducing bugs.
+                It is way easier to resolve a bug report with a code example and
+                that is verified by someone.
+            </p>
+
+            <p>
+                Have a look at the <a href="https://github.com/symfony/symfony/issues?q=is%3Aissue+is%3Aopen+label%3ABug">open bug reports</a>,
+                try to understand and reproduce the bugs. Make a comment like
+                "Verified" or "Cannot reproduce". Also explain what you have tried,
+                maybe add some code example or maybe event a link to a new test repository.
+            </p>
+
+            <p>
+                If the bug was verified and you want to walk the extra mile,
+                you can also try fixing it. Ask in Slack for help.
+            </p>
+
+            <h4>Review open PRs</h4>
+            <p>
+                If you really want a challenge, go the the <a href="https://github.com/symfony/symfony/pulls?page=2&q=is%3Apr+is%3Aopen+label%3A%22Status%3A+Needs+Review%22">open PRs</a>
+                and review them. Make a comment about what you like and what could
+                be improved. Maybe event test the PR to make sure it works. Then try
+                to figure out why the PR is still open.
+            </p>
+            <ul>
+                <li>Does it need a decision for the Core Team?</li>
+                <li>Does it need more work? If the PR has been stalled for a long time, maybe you want to finish it?</li>
+                <li>If it is not clear why it is stalled, make sure to submit a Github review with "Approve" or "Request changes"</li>
+            </ul>
+
+            <p>How to checkout someone else's PR?</p>
+            <code><pre>
+git fetch origin pull/38920/head:pr-38920
+git checkout pr-38920
+            </pre></code>
+
+            <h4>Tell us about testing</h4>
+            <p>
+                TODO
+            </p>
+            <h4>Write a blog post</h4>
+            <p>
+                An absolutely awesome way to contribute to symfony is to write a
+                blog post where you share an idea or describe how you solved a problem.
+                Even if a problem is very specific, it will still be helpful for
+                some people.
+            </p>
+            <p>
+                Blog posts differs from the Symfony documentation because they can
+                be opinionated. There are sometimes mor than one way to solve a
+                problem and none is "the best". The documentation is also very restrictive
+                what libraries to recommend.
+            </p>
+            <p>
+                If you write a blog post, make sure to spread it on Twitter, in the
+                #posts channel on Symfony Devs slack and on <a href="https://dev.to/">dev.to</a>.
```

Comment:
I won't add "dev.to" here. Imho, it's completely up to the writer were to post it (e.g. Medium is just as fine as dev.to, only in #posts and twitter also).

In fact, I think "make sure to share it in the #posts channel on Symfony slack and any other social media." would be better, don't you think?

Reply:
Sure.